src/util.rs

```diff
@@ -83,3 +83,35 @@ fn test_translationtable_translate() {
         "This h is a helicopter!".to_owned(),
     )
 }
+
+// Tries to guess the encoding, not perfect but does it's job
+pub fn decode_bytes<Bytes: Into<Vec<u8>>>(bytes: Bytes) -> Option<String> {
+    let bytes = bytes.into();
+    let (label, confidence, _) = chardet::detect(&bytes);
```

Comment:
I don't know if we need `chardet`, why not use `for_label` from `encoding_rs` to get the encoding?

Reply:
`from_label` expects an encoding label, `chardet` gets us a label for the encoding

Reply:
Is `from_label` a typo? I mean https://docs.rs/encoding_rs/0.8.23/encoding_rs/struct.Encoding.html#method.for_label, I thought it takes in the original data?

Reply:
I haven't used that method before, but yeah, it _looks_ like it's just the parser logic for a label, which we won't have.

Reply:
Yes `from_label` was a typo, `for_label` expects the label, not the data, `detect` gets the label from the data